runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json

```diff
@@ -329,6 +329,12 @@
       "icon": 21978,
       "name": "Extended Super Antifire (4)",
       "xp": 160
+    },
+    {
+      "level": 92,
```

Comment:
This should be placed between super combat and anti-venom potions so it is in ascending level order.